cyclus/system.py

```diff
@@ -34,7 +34,7 @@ def asyncio():
 
 @lazyobject
 def websockets():
```

Comment:
This should look like the curio replacement above, but replace curio with websockets.